docs/docs/cheat-sheet.md

```diff
@@ -0,0 +1,283 @@
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/tutorial">gatsby.dev/tutorial</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Quick Start<br />(for intermediate and advanced developers)</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/quick-start">gatsby.dev/quick-start</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Gatsby Starters</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/starters">gatsby.dev/starters</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Quick Reference Guide</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/recipes">gatsby.dev/recipes</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Adding Images</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/image">gatsbyjs.dev/image</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Gatsby Node APIs</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/api">gatsby.dev/api</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Querying with GraphQL</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/graphql">gatsby.dev/graphql</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Deploying and Hosting</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/deploy">gatsby.dev/deploy</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Using Gatsby Link</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/link">gatsby.dev/link</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>Static Query</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/static-query">gatsby.dev/static-query</a></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <p>How to Contribute</p>
+                </td>
+                <td>
+                    <p><a href="https://gatsby.dev/contribute">gatsby.dev/contribute</a></p>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+    <p><a href="https://www.gatsbyjs.org/">gatsbyjs.org</a></p>
+    <p><a href="https://twitter.com/gatsbyjs">gatsbyjs</a></p>
```

Comment:
should be visible that it is a link to twitter

Reply:
This entire table is visually hidden and acting as alt text for the graphics on the page–I don't think that is necessary.

Reply:
But alt text is visible for screen reader? Then we should fix it ...

Reply:
```suggestion
    <p><a href="https://twitter.com/gatsbyjs">twitter.com/gatsbyjs</a></p>
```